packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -51,7 +51,7 @@
   }
 
   .file-input-target {
-    border: 1px dashed var(--uswds-system-color-gray-30);
```

Comment:
We should not have been using a uswds system color token to begin with. I'm actually not sure that this should even be accessible from the css-library.